style.scss

```diff
@@ -105,36 +105,46 @@ header {
 main {
     flex-grow: 1;
 
-    .work {
+    .hero-section {
+        padding: 10px;
         background-image: url("./img/man.jpg");
-        background-size: cover;
         background-position: center;
-        height: 400px;
+        background-size: cover;
+        height: 300px;
 
         @media (min-width: $screen-md) {
-            height: 600px;
+            height: 400px;
+        }
+
+        @media (min-width: $screen-lg) {
+            height: 500px;
         }
     }
 
-    .text-about-us {
+    .motto {
         text-align: center;
-        padding-top: 100px;
-        margin: 0 300px 0 10px;
+        padding-left: 1rem;
+        text-shadow:
+            1px 1px 2px $dark,
```

Comment:
Good stuff! The only thing I want is to keep units consistent. Please don't mix them, use either pixels, or, ems. Better to use rems in this case.